src/cfe.c

```diff
@@ -180,7 +185,15 @@ extern void cfe(
   
   conceptual_reservoir_flux_calc(gw_reservoir_struct,&primary_flux,&secondary_flux);
   
+  
   flux_from_deep_gw_to_chan_m=primary_flux;  // m/h   <<<<<<<<<< BASE FLOW FLUX >>>>>>>>>
+  if(flux_from_deep_gw_to_chan_m >  gw_reservoir_struct->storage_m)  {
+  flux_from_deep_gw_to_chan_m=gw_reservoir_struct->storage_m;

```

Comment:
What happens to the excess flux (`gw_reservoir_struct->storage_m - flux_from_deep_gw_to_chan_m`)?

Reply:
Nothing has to happen to it. We cannot remove more than it is available in the storage, even if the equation that calculates the flux (which is conceptual) tells us to. So we end up removing ONLY what is available. That should not create any issues with water balance.

Reply:
My point is _something_ had happened to it. My only recommendation is to add an `error` accumulator to put these types of errors into so you can see them along side other diagnostic info.

Reply:
The flux in line 189 is like a "potential flux" not a real flux. It is calculated based on a conceptual equation (you can even think of it like a "regression"). This flux is the real flux if enough water is available in the reservoir. if not, then the real flux is limited to the amount of water in the storage (line 191). So it is not an error. It does not interfere with the water balance. There is nothing that physically obligates the flux to be equal to what was calculated by the conceptual equation.

Reply:
Gotcha. It was a little hard to see just based on the diff I was reading. But that is why reviews are useful, just in case and just to learn a little 😁